.github/workflows/sanitycheck_tests.yml

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2020 Linaro Limited.
+# SPDX-License-Identifier: Apache-2.0
+
+name: Zephyr West Command Tests
+
+on:
+  push:
+    paths:
+    - 'scripts/sanity_chk/sanitylib.py'
+    - 'scripts/sanitycheck'
+    - 'scripts/tests/sanitycheck/**'
+  pull_request:
+    paths:
+    - 'scripts/sanity_chk/sanitylib.py'
+    - 'scripts/sanitycheck'
+    - 'scripts/tests/sanitycheck/**'
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+        os: [ubuntu-latest]
+    steps:
+    - name: Run sanitycheck testsuite
+      run: |
+        echo "Run Sanitycheck tests"
+        pytest $ZEPHYR_BASE/scripts/tests/sanitycheck
```

Comment:
does the environment actually have pytest? What about all the python packages sanitycheck needs? There seems to be a lot missing here to make this work